ci/test_cudf_polars.sh

```diff
@@ -28,11 +28,8 @@ rapids-logger "Install cudf wheel"
 # echo to expand wildcard before adding `[extra]` requires for pip
 python -m pip install $(echo ./dist/cudf*.whl)[test]
 
-rapids-logger "Install polars (allow pre-release versions)"
-python -m pip install 'polars>=1.0.0a0'
-
 rapids-logger "Install cudf_polars"
-python -m pip install --no-deps python/cudf_polars
+python -m pip install python/cudf_polars
```

Comment:
This is annoying, but right now we have to do this because the cudf package will be `-cu11/12`. We should figure out how we want this to look going forward, but since this PR is now blocking all pylibcudf/polars PRs I'm just suggesting the minimal fix for now.
```suggestion
python -m pip install `polars>=1.0`
python -m pip install --no-deps python/cudf_polars
```

Reply:
Ah this is the same problem as is going on with the ucx-py source packages as well...